README.md

```diff
@@ -24,6 +27,18 @@ Inizialmente, il progetto è stato sviluppato in *javascript* ma in una sua seco
 
 Il framework utilizzato nel progetto è ***React*** ([qui](https://it.reactjs.org/) la documentazione ufficiale): l'allenamento nell'utilizzo di queste librerie è stato il motivo principale della nascita di questo esercizio pratico.
 
+## YARN
+
+In questa sezione viene mostrato cosa fare **dopo** aver clonato il progetto.
+
+### INSTALL
+
+Lanciare il comando *yarn install* per installare tutte le dipendenze del progetto. In caso di successo, si può notare la creazione della cartella *node_modules* (si consiglia di escludere questa cartella dal git).
```

Comment:
utilizzare snippet per i comandi. "dal git" non è linguaggio tecnico preciso, sostituire con "escludere la cartella aggiungendola nel .gitignore". Anche se la cosa è migliore è rimuovere la frase dato che è la folder è già esclusa

Reply:
Frase eliminata in un commit successivo